selfdrive/car/interfaces.py

```diff
@@ -99,8 +99,13 @@ def get_params(cls, candidate: str, fingerprint: Dict[int, Dict[int, int]], car_
     ret = CarInterfaceBase.get_std_params(candidate)
     ret = cls._get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs)
 
-    # Set params dependent on values set by the car interface
+    # Set common params using fields set by the car interface
+    # TODO: get actual value, for now starting with reasonable value for
+    # civic and scaling by mass and wheelbase
     ret.rotationalInertia = scale_rot_inertia(ret.mass, ret.wheelbase)
+
+    # TODO: start from empirically derived lateral slip stiffness for the civic and scale by
+    # mass and CG position, so all cars will have approximately similar dyn behaviors
```

Comment:
The TODOs do have value, but I pulled them as cleanup: they duplicate comments in the called functions. It would be reasonable to pull [the copies in `__init__.py`](https://github.com/commaai/openpilot/blob/3b849811e5298a59cc723937ec94a458fd6dad88/selfdrive/car/__init__.py#L55-L71) if you prefer to keep them in this location.

Reply:
I see, nice catch!